### ISSUE TITLE
"Same texture with different mask inserted" Error

### ISSUE THREAD

**Issue by handicast** [Taranchuk/FasterGameLoading#2]
https://gist.github.com/HugsLibRecordKeeper/81f127203660ee64be56a69ad98a51e0
I only have the first and last option on. This causes the texture to become a white square for some reason. The 2 middle options completely bug my game and cause this problem to be even more apparent, making pawns and literally everything a white square.

https://gist.github.com/HugsLibRecordKeeper/aaf5c087d65adef6f399f13f0ac5b16a
^ This is a log without FGL on. There are no texture errors.

**Comment by SirHumphryApplebee** [comment]
So far I've tracked it down to Bionic Icons and XenoBionicPatcher.

https://gist.github.com/HugsLibRecordKeeper/260792d0010fd52271efea3090fa5300

**Comment by p87-lyng** [comment]
This might be an old issue - but I concur that this error is being thrown in large amounts in 1.5 as well, with Bionic Icons enabled.

**Comment by Sakurasrb** [comment]
> This might be an old issue - but I concur that this error is being thrown in large amounts in 1.5 as well, with Bionic Icons enabled.

True, this mod seems to be imcompatible with Bionic Icons when the second option is turned on.

**Comment by D3nnis3n** [comment]
This issue is still present in RimWorld 1.6 with respectively updated mods as well.
